src/components/GoalCriteria/GoalCriteria.module.css

```diff
@@ -0,0 +1,32 @@
+.GoalCriteriaItemTitle {
+    flex: 1;
+}
+
+.GoalCriteriaIssueMeta {
+    display: grid;
+    grid-template-columns: 100%;
+    grid-template-rows: minmax(32px, 100%);
+    align-items: center;
+    gap: var(--gap-s);
+    max-width: 520px;
+}
+
+.GoalCriteriaCheckBox {
+    display: inline-flex;
+    cursor: pointer;
+    color: var(--gray-800)
+}
+
+.GoalCriteriaCheckBox:hover {
+    color: var(--gray-500)
+}
+
+.GoalCriteriaCheckBox_disabled {
+    pointer-events: none;
+    cursor: default;
+}
+
+.GoalCriteriaCheckBox_checked, .GoalCriteriaCheckBox_checked:hover{
```

Comment:
Add space before bracket
Also would be better to write list of selectors on column, not in a row